_test/test_gen_sqw_workflow/test_gen_sqw_accumulate_sqw_herbert.m

```diff
@@ -51,7 +51,7 @@
             end
             combine_algorithm = 'mpi_code'; % this is what should be tested
             if is_jenkins && ispc
-                combine_algorithm = 'mex_code'; % disable mpi combine on Jenkins. It is extremely slow.
+                combine_algorithm = 'mex_code'; % disable poor man MPI combine on Jenkins. It is extremely slow.
```

Comment:
We currently call this a mix of "poor man MPI" and "ClusterHerbert"? Should we settle on a name?

Reply:
yes, these two names used intermittently.  Do not know if we should have one -- the second is the real name of the class, which implements "poor man MPI" algorithm. Not sure one name covers all -- one is for class, other describes the essence of the algorithm.